src/site/stages/build/plugins/modify-dom/check-broken-links/index.js

```diff
@@ -83,6 +83,7 @@ module.exports = {
       brokenLinksCount,
     };
 
+    fs.ensureFileSync(this.logFile);
```

Comment:
https://dsva.slack.com/archives/C01512KN35G/p1619729648007200?thread_ts=1619706603.480000&cid=C01512KN35G

Reply:
We need this to ensure the `logs` dir exists before trying to write the file.